mainwindow.cpp

```diff
@@ -10,9 +10,15 @@
  */
 
 #include "mainwindow.h"
-#include "connectdialog.h"
 #include "ui_mainwindow.h"
+#include "connectdialog.h"
 #include "sessionmanager.h"
+#include "outputmanager.h"
+
+#include <QTextEdit>
```

Comment:
Questo non dovrebbe servirti, non usi `QTextEdit` da nessuna parte in questo modulo.